cmd/logs/logs.go

```diff
@@ -35,7 +35,11 @@ func Start() {
 	l := listener.New(config.GetLogsSources(), pp)
 	l.Start()
 
-	s := tailer.New(config.GetLogsSources(), pp, a)
+	tailingLimit := LogsAgent.GetInt("wildcard_open_files_limit")
```

Comment:
should be `config.LogsAgent` instead

Reply:
also, by convention the way we generally set defaults on a configuration option is by using `SetDefault` directly on the config struct (see https://github.com/DataDog/datadog-agent/blob/6.0.0-beta.6/pkg/config/config.go#L59-L153). In the logs code it could make sense to do that in `config.buildMainConfig`

Reply:
and finally: if this setting is meant to be changed by users it should be documented in the `datadog.yaml.example` config template here: https://github.com/DataDog/datadog-agent/blob/6.0.0-beta.6/pkg/config/config_template.yaml

It should probably be prefixed with `log_` or something similar to make it clear that it affects only the logs part of the agent. And I'm not sure `wildcard` really applies to the option anymore (the limit on the number of open files is enforced on log files that are not specified with wildcards, correct?), so a name like `log_open_files_limit` would make more sense to me.

Reply:
Agreed, thanks for your feedback, I made all the changes,